subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_dependencies.adoc

```diff
@@ -1,6 +1,8 @@
 [[declaring-dependencies]]
 = Declaring dependencies
 
+Before looking at dependency declarations themselves, the concept of _dependency configuration_ needs to be defined.
+
```

Comment:
Not a comment for this PR, but in general:

I think we should consider to talk about variants here first and then go from there explaining how they are built up using "configurations". This would also make this chapter more use case oriented.